modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -1,29 +1,41 @@
 <template>
-  <div class="apos-field__outer">
+  <div class="apos-field__wrapper">
     <component :is="wrapEl" :class="classList">
-      <!-- TODO i18n -->
-      <component
-        v-if="field.label" :class="{'apos-sr-only': field.hideLabel }"
-        class="apos-field-label"
-        :is="labelEl" :for="uid"
-      >
-        {{ field.label }}
-        <span v-if="field.required" class="apos-field-required">
-          *
-        </span>
-      </component>
-      <!-- TODO i18n -->
-      <p v-if="field.help" class="apos-field-help">
-        {{ field.help }}
-      </p>
-      <p
-        v-else-if="field.htmlHelp" class="apos-field-help"
-        v-html="field.htmlHelp"
-      />
-      <slot name="additional" />
+      <div class="apos-field__info">
+        <!-- TODO i18n -->
+        <component
+          v-if="field.label" :class="{'apos-sr-only': field.hideLabel }"
+          class="apos-field__label"
+          :is="labelEl" :for="uid"
+        >
+          {{ field.label }}
+          <span v-if="field.required" class="apos-field__required">
+            *
+          </span>
+          <span
+            v-if="(field.help || field.htmlHelp) && displayOptions.helpTooltip"
+            class="apos-field__help-tooltip"
+          >
+            <AposIndicator
+              icon="help-circle-icon"
+              class="apos-field__help-tooltip__icon"
+              :tooltip="(field.help || field.htmlHelp)"
+              :icon-size="11"
+              icon-color="var(--a-base-4)"
+            />
+          </span>
+        </component>
+        <!-- TODO i18n -->
+        <p
+          v-if="(field.help || field.htmlHelp) && !displayOptions.helpTooltip"
+          class="apos-field__help"
+          v-html="(field.help || field.htmlHelp)"
```

Comment:
@abea do note the consolidation of this help UI, its new in the most recent merge (tested and working to put HTML into the tooltip bubble as well)